portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,10 +82,49 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         //response[1] = capitals.get(4);
         Random rand = new Random();
 
-        response.setContentType("text/html;");
-        response.getWriter().println(countries.get(rand.nextInt(countries.size())));
+        // response.setContentType("text/html;");
+        // response.getWriter().println(countries.get(rand.nextInt(countries.size())));
+        String json3 = "{";
+        json3 += "\"gender\": ";
+        json3 += "\"" + "Male" + "\"";
+        json3 += ", ";
+        json3 += "\"height\": ";
+        json3 += "\"" + "1.75" + "\"";
+        json3 += "}";
+
+        String json = convertArrayListToJson(hardCodedJson);
+        // String json = convertToJsonUsingGson(hardCodedJson);
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
     }
 
+    public String convertArrayListToJson(List<String> list) {
+        // Starting json string
+        String json = "{";
+        json += "\"messages\": [";
+        
+        // Adding all hard-coded messages
+        for (int i = 0; i < list.size()-1; i++) {
```

Comment:
Java has a simpler loop structure that I would prefer for all cases where the index (`i`) isn't needed.

```
for (String item : list) {
 ...
}
```

Reasoning: While relatively simple, the for loop could have subtle bugs (off by one in the end condition, `i` could get reset in the middle of the loop, etc.). Using the simplest possible way of doing something means that it's easier to review and analyze.